apps/system/js/notifications.js

```diff
@@ -403,44 +402,57 @@ var NotificationScreen = {
      */
     var dir = (detail.bidi === 'auto' || typeof detail.bidi === 'undefined') ?
       document.documentElement.dir : detail.bidi;
-
+    // As per the UX,in case for sms notification ,the notification title
+    // will be the count along with the 'new message' text,but currently in
+    // 'detail' object, name of the sender is coming. in other cases title text
+    // in 'detail' object is appropriate.
+    var cIndex = this.unReadNotificationIndex(manifestURL);
+    if (cIndex === -1) {
```

Comment:
"It looks like the closing brace for this if clause is 60 lines later! And you haven't indented the intervening 60 lines correctly, so the code becomes very hard to read." (C) David from PR https://github.com/mozilla-b2g/gaia/pull/32749.
Please, in such cases ((un)wrapping of existent code with ``if`` or ``switch`` or whatever) don't forget to properly indent such blocks and if nothing except indentation is changed in such cases, you can just put comment in PR mentioning this, to speed up review.